resources/puppetlabs/puppetdb/pql/pql-grammar-experimental.ebnf

```diff
@@ -125,6 +123,15 @@ sqstring = (<singlequote>, stringwithoutsinglequotes, <singlequote>);
 <modifier> = groupby;
 groupby  = <'group by '>, fieldlist;
 
+(* Paging *)
```

Comment:
I can't see any parser tests for this new block. Probably worth adding since parts of this are squirrely.